app/helpers/meeting_contents_helper.rb

```diff
@@ -49,29 +49,71 @@ def meeting_agenda_toggle_status_link(content, content_type)
   def close_meeting_agenda_link(content_type, meeting)
     case content_type
     when 'meeting_agenda'
-      link_to_if_authorized l(:label_meeting_close), { controller: '/meeting_agendas', action: 'close', meeting_id: meeting }, method: :put, class: 'icon icon-locked show-meeting_agenda'
+      content_tag :div, '', class: 'button' do
```

Comment:
Those `<div class="button">` elements probably should be `<button class="button">`. At least this is whats used in the toolbar.

Which leads me to the main question: Couldn't you use the toolbar component here instead of manually building something that looks like a toolbar?

There is a `ToolbarHelper`: https://github.com/opf/openproject/blob/dev/app/helpers/toolbar_helper.rb
An example usage can be found here: https://github.com/opf/openproject/blob/dev/app/views/projects/index.html.erb#L35-L47